### ISSUE TITLE
jepsen slow-net and stress tests beaking other :batch tests

### ISSUE THREAD

**Issue by vladzcloudius** [scylladb/jepsen#3]
HEAD: 692e5b899cac1b7f05678d614c8291bdd3008801

There is a bug in the :batch :slow-net tests that breaks other :batch tests: when run after the corresponding :slow-net test the original test runs no transaction at all. The side effect is an empty history.txt file.

**Comment by vladzcloudius** [comment]
The root cause was that I used the `def` for creating an "original" test (e.g. `bridge-test`) and then used it in order to create a new `slow-net` test (`bridge-test-slow-net` for the example above).
The problem is that the test is not just a test - it's a test state and it in particular these tests have some mutable parts like `:bootstrap` fields. Therefore it may not be used like I did. 
One way to fix it is to create the actual state as the last step of the test definition chain and make all previous steps functions. This way we may both reuse the existing code and make all tests' states' separate.
